src/XrdOssStats/XrdOssStatsDirectory.hh

```diff
@@ -0,0 +1,45 @@
+
+#ifndef __XRDSTATS_DIRECTORY_H
+#define __XRDSTATS_DIRECTORY_H
+
+#include "XrdOuc/XrdOucEnv.hh"
+#include "XrdOss/XrdOssWrapper.hh"
+#include "XrdOssStatsFileSystem.hh"
+#include "XrdSys/XrdSysError.hh"
+
+#include <memory>
+
+class StatsDirectory : public XrdOssWrapDF {
```

Comment:
OK, I see you did move the plugins into XrdOssStats directory. This always forms the basis of subsequent names. For instance, namespace XrdOssStats { class Directory...} which is the closest you can come here. Normally, classes would be named XrdOssStatsDirectory for consistency. However, namespaces to keep the naming consistent are also acceptable. Why do we do this? To ease debugging across multiple compilation units by using a consistent naming scheme. So, I would suggest jiggering the names here.

Reply:
Rename is done in the latest patch.